stdlib/private/StdlibUnittest/StdlibUnittest.swift.gyb

```diff
@@ -1437,6 +1438,8 @@ public enum OSVersion : CustomStringConvertible {
       return "PS4"
     case .android:
       return "Android"
+    case .windows:
+      return "Windows"
```

Comment:
For this change to StdlibUnittest, would you mind adding a test like `validation-test/StdlibUnittest/Linux.swift`?